### PR TITLE
Add Evaluation Result Asset folders and files

### PR DESCRIPTION
**Pull request by YunsongB**
Add Evaluation Result Asset folders and files.

### CODE REVIEW

**Comment by alexkalita** [comment]
Can you add "categories:" entries to the asset.yaml files?  There is a gated test that is checking for these:

Error: Validation of assets/evaluation_results/mmlu_other_gpt_4_0314_question_answering/asset.yaml failed: Categories not found

For reference the categories are used to populate the documentation pages: https://github.com/Azure/azureml-assets/wiki/models-documentation

**Comment by YunsongB** [comment]
> Can you add "categories:" entries to the asset.yaml files? There is a gated test that is checking for these:
> 
> Error: Validation of assets/evaluation_results/mmlu_other_gpt_4_0314_question_answering/asset.yaml failed: Categories not found
> 
> For reference the categories are used to populate the documentation pages: https://github.com/Azure/azureml-assets/wiki/models-documentation

Added.

**Comment by YunsongB** [comment]
> Can you add "categories:" entries to the asset.yaml files? There is a gated test that is checking for these:
> 
> Error: Validation of assets/evaluation_results/mmlu_other_gpt_4_0314_question_answering/asset.yaml failed: Categories not found
> 
> For reference the categories are used to populate the documentation pages: https://github.com/Azure/azureml-assets/wiki/models-documentation

Added.

**Review by yuanzhuangyuanzhuang — DISMISSED** [review]

File: assets/evaluation_results/boolq_gpt_35_turbo_0301_question_answering/spec.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+type: evaluationresult
+name: boolq_gpt_35_turbo_0301_question_answering
+version: 1.0.0
+display_name: boolq_gpt_35_turbo_0301_question_answering
+description: gpt-35-turbo-0301 run for boolq dataset
+dataset_family: boolq
+dataset_name: boolq
+
+model_name: gpt-35-turbo-0301
+model_version: "301"
```
Comment (EricWrightAtWork) [inline assets/evaluation_results/boolq_gpt_35_turbo_0301_question_answering/spec.yaml:10]: Not critical - but these versions strings really should be "0301" and "0314" to match the way that OAI/AOAI versions these models

**Review by alexkalita — APPROVED** [review]

**Review by jialiu103 — APPROVED** [review]

File: assets/evaluation_results/boolq_gpt_35_turbo_0301_question_answering/spec.yaml (hunk 1 above)
Comment (YunsongB) [reply assets/evaluation_results/boolq_gpt_35_turbo_0301_question_answering/spec.yaml:10]: Will change them in the next release.

**Review by vizhur — DISMISSED** [review]

**Review by alexkalita — APPROVED** [review]

**Review by EricWrightAtWork — APPROVED** [review]

**Review by yuanzhuangyuanzhuang — APPROVED** [review]

**Review by jialiu103 — APPROVED** [review]

**Review by jialiu103 — APPROVED** [review]

**Review by vizhur — APPROVED** [review]

**Pull request merged**